eigenfactor.js

```diff
@@ -161,7 +145,7 @@ function flowEdges(links) {
   return l.map(e => {
     if ("weight" in e) {
       e.normalizedWeight =
-        Math.log(1 + (e.weight / maxEdgeWeight) * 10) / Math.log(11);
+        Math.log(1 + e.weight / maxEdgeWeight * 10) / Math.log(11);
```

Comment:
We have a different version of prettier :)

Reply:
Oh no! Which version do you run? We can also add a .prettierrc file if you like.

Reply:
OK, I updated to 1.14.0, hope that fixes it.